api/src/pcapi/core/search/backends/algolia.py

```diff
@@ -526,6 +526,7 @@ def serialize_collective_offer(cls, collective_offer: educational_models.Collect
                 else None,
                 "eventAddressType": collective_offer.offerVenue.get("addressType"),
                 "beginningDatetime": beginning_datetime,
+                "description": remove_stopwords(collective_offer.description or ""),
```

Comment:
Le champ `CollectiveOffer.description` n'est pas nullable, on devrait pouvoir écrire : 

```suggestion
                "description": remove_stopwords(collective_offer.description),
```

Idem plus bas pour les templates.


(c'est d'ailleurs ce que j'avais proposé dans mon commentaire initial)

Reply:
> (c'est d'ailleurs ce que j'avais proposé dans mon commentaire initial)

oops oui désolé :fearful: 
Je corrige ! 

(merci !)